expences-app-swiftui/Views/ExpenseEditingView.swift

```diff
@@ -0,0 +1,105 @@
+//
+//  AddExpense.swift
+//  expences-app-swiftui
+//
+//  Created by Serhii Prykhodko on 16.10.2020.
+//
+
+import SwiftUI
+
+struct ExpenseEditingView: View {
+    
+    @Binding var expense: Expense?
```

Comment:
these properties are all put here in random order, please consider grouping them (binding/state etc) and sorting each group for better readability